docs/operator_subscriptions/subscriptions.md

```diff
@@ -0,0 +1,190 @@
+# Rucio Subscriptions
+
+## Overview
+
+A **subscription** in Rucio defines a set of rules and filters that are automatically applied to datasets and files as they are registered in the system. Subscriptions can trigger the creation of replication rules for matching data, enabling automated data placement and management.
+
+Subscriptions are a core way to automate data workflows and ensure datasets are distributed according to organizational needs.
+
+---
+
+## Why Use Subscriptions?
+
+- **Automate Data Management:** Replicate datasets automatically based on metadata, names, or other attributes.
+- **Policy Enforcement:** Ensure compliance with data distribution policies.
+- **Efficiency:** Avoid manual rule creation for every new dataset.
+
+---
+
+## Key Concepts
+
+- **Filter:** A dictionary specifying criteria for which data the subscription applies to (e.g., dataset name patterns, account, project).
+- **Replication Rules:** Instructions for how matching data should be replicated (number of copies, target RSEs, weights, etc).
+- **Retroactive:** Whether the subscription should apply to pre-existing data.
+- **Lifetime:** Optional duration (in days) for the subscription to remain active.
+- **Priority:** Determines the order in which subscriptions are processed.
+- **State:** Subscriptions can be ACTIVE, NEW, INACTIVE, or UPDATED.
```

Comment:
Can also be `BROKEN` for what I see in `db/sqla/constants`, not sure if we also mention this.